Makefile

```diff
@@ -70,3 +71,12 @@ beats-dashboards:
 .PHONY: docs
 docs:
 	sh libbeat/scripts/build_docs.sh ${PROJECTS}
+
+.PHONY: package
+package:
+	$(MAKE) -C libbeat package-setup
+	$(foreach var,$(BEATS),SNAPSTHOT=$SNAPSHOT $(MAKE) -C $(var) package || exit 1;)
```

Comment:
Would be good if either here or in the beats Makefile it would print some messages like `Start building packages for ${BEAT}` and `Finished packages for ${BEAT}`. The unified build process can use these to print show some progress indications.